content/blog/2025-05-07-tmil-17/index.md

```diff
@@ -0,0 +1,123 @@
++++
+title = "Linebender in May 2025"
+authors = ["Olivier Faure"]
++++
+
+Linebender is an informal open-source organization working on various projects to advance the state of the art in GUI for [the Rust programming language](https://rust-lang.org).
+
+
+## RustWeek
+
+This month saw the [RustWeek 2025](https://rustweek.org/) conference, in Utrecht in the Netherlands.
+
+A lot of things happened!
+
+### The talks
+
+Raph Levien gave a talk titled *Faster, easier 2D vector rendering*, about Vello's new architecture.
+<!-- TODO: Add link to Youtube video of the talk. -->
+<!-- If last year is any indication, it should be uploaded soon. -->
+The talk had some nice visuals and a pretty cool demo!
+We recommend you check it out.
+<!-- TODO: Add screenshot of Raph's demo. -->
+
+Matt Campbell gave a talk titled *AccessKit: reusable UI accessibility*, showing off AccessKit and its API and architecture.
+The talk also came with a cool demo, showcasing AccessKit's Android integration.
+
+### The Anniversary
+
+RustWeek also [the release of Rust 1.87](https://blog.rust-lang.org/2025/05/15/Rust-1.87.0/), which also fell exactly on the 10 year anniversary for Rust 1.0!
+
+(If you're like me and you're wondering how that works: Rust has one release every six weeks. `87 * 6 * 7 == 3654` and `3654 == 365 * 10 + 4`. Between 2015 and 2025, there have been four bissextile years: 2016, 2020, 2024, and... wait a minute. Well, turns out Rust 1.1 came out one day earlier than the 6-weeks schedule implies, June 25 2015 instead of June 26 2015. There may have been other discrepancies, I haven't checked all the dates.)
```

Comment:
s/bissextile/leap/? I didn't know the word.

Reply:
Eh, I think I'll leave it. It's a nerdy aside anyway.